projects/arlas-toolkit/src/lib/services/settings/arlas.settings.service.ts

```diff
@@ -60,6 +60,14 @@ export class ArlasSettingsService {
     return !!this.settings && !!this.settings.histogram && !!this.settings.histogram.max_buckets ? this.settings.histogram.max_buckets : 200;
   }
 
+  public isResultListExportEnabled(): boolean {
+    return !!this.settings && !!this.settings.resultlist && !!this.settings.resultlist.enable_export;
+  }
+
+  public getResultlistSettings(): ResultlistSettings {
+    return this.isResultListExportEnabled() ? this.settings.resultlist : undefined;
```

Comment:
Why would we be able to get the settings of the list only if it is exportable ?

Reply:
fixed